src/test/java/org/kcctl/command/GetConnectorsCommandTest.java

```diff
@@ -68,7 +68,7 @@ public void should_print_registered_connectors() {
         }))
                 .map(String::trim)
                 .containsExactly(
-                        "NAME    TYPE     STATE     TASKS",
```

Comment:
It looks like we're falling prey to https://github.com/freva/ascii-table/issues/21 here, where ANSI escape codes are being treated as visible non-zero-width characters by the ASCII table library when determining how to space out its headers.

Could we use the `Styler` interface introduced in https://github.com/freva/ascii-table/commit/0cd488748e7797fe5cc7ef1ae1cd41f9641affec to add color to state strings after the table's dimensions have already been determined?

I was initially tempted to suggest sticking with the current `String::replace` approach, but that fails when connectors' names match state names.